### PR TITLE
crypto/ec/curve448/eddsa.c: fix EBCDIC platforms

### PR DESCRIPTION
**Pull request by p-steuer**

### CODE REVIEW

**Comment by p-steuer** [comment]
travis red cross due to "make: *** [doc-nits] Error 1" looks unrelated to me

File: crypto/ec/curve448/eddsa.c [hunk 1]
```diff
@@ -58,7 +58,11 @@ static c448_error_t hash_init_with_dom(OPENSSL_CTX *ctx, EVP_MD_CTX *hashctx,
                                        const uint8_t *context,
                                        size_t context_len)
 {
-    const char *dom_s = "SigEd448";
+#ifdef CHARSET_EBCDIC
+    const char dom_s[] = {0x53,0x69,0x67,0x45,0x64,0x34,0x34,0x38,0x00};
```
Comment (mattcaswell) [inline crypto/ec/curve448/eddsa.c]: nit: space after the ","

**Comment by mattcaswell** [comment]
Travis red cross appears to be an issue in master (!!) and not related to this PR

**Comment by p-steuer** [comment]
Fixed and squashed nit.

**Comment by p-steuer** [comment]
merged.
1.1.1:https://github.com/openssl/openssl/commit/ded6741604e7f622dc278f6b24c0b47432b75325
master: https://github.com/openssl/openssl/commit/af7f656cd91d99d62567e2b20c61f07cb4d81d0b

**Pull request closed without merging**